ocs_ci/framework/conf/default_config.yaml

```diff
@@ -32,6 +32,7 @@ RUN:
   # This config file disables scale app pods to use OCS workers
   use_ocs_worker_for_scale: False
   load_status: None
+  collect_logs_on_success_run: False
```

Comment:
I see here:
https://github.com/red-hat-storage/ocs-ci/pull/2880/files#diff-153ccf5916ca230718b5455c349afbcdR126

we have it in reporting section, should we consider it to have it in reporting section as well here instead of RUN section?

Reply:
I think yes, I just followed the rule from some other argument that utilizes the logs.
Will move

Reply:
I am thinking of taking off the `_run` from the argument name before we merge it. @petr-balogh